library/Zend/Filter/StringToLower.php

```diff
@@ -53,7 +53,7 @@ public function filter($value)
         }
         $value = (string) $value;
 
-        if ($this->options['encoding'] !== null) {
+        if (null !== $this->getEncoding()) {
```

Comment:
the #3521 and #3520 , the yoda condition is not accepted. anyway, the current purpose of this pr is changing <code>$this->options['encoding']</code> to <code>$this->getEncoding()</code>, so changing right to left is a cosmetic imo